### DISCUSSION TITLE
newest version of react-aria-components 'client-only' use in server component

### DISCUSSION THREAD

**Discussion by skusez** [adobe/react-spectrum#5880] [Q&A]
calling all nextjs users 🤓

Before the latest update, I could directly use imported components in my server files. So things like Text, Heading that I wouldnt bother creating a "use client" wrapper for would just work when imported to a server file

Now I need to wrap every component in a use client file? seems a bit cumbersome and a step in the wrong direction. 

Could someone shed some light on this decision? I love this library and Im sure the team has good reason for this change ❤️

eg:

```jsx
import {Text} from "react-aria-components"

export const MyPage = () => {
return (
<div>
<Text>Some text</Text>
</div>
)
}
```

meets this error:
```
'client-only' cannot be imported from a Server Component module. It should only be used from a Client Component.
The error was caused by importing 'react-aria-components/dist/import.mjs'
```

**Comment by snowystinger** [comment]
Please see https://github.com/adobe/react-spectrum/pull/5826 for more details. The change should improve your bundle size.
